presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/query/PinotFilterExpressionConverter.java

```diff
@@ -89,15 +100,109 @@ private PinotExpression handleLogicalBinary(
         }
         List<RowExpression> arguments = call.getArguments();
         if (arguments.size() == 2) {
-            return derived(format(
-                    "(%s %s %s)",
-                    arguments.get(0).accept(this, context).getDefinition(),
-                    operator,
-                    arguments.get(1).accept(this, context).getDefinition()));
+            // Trying to pushdown the predicate of comparing a time column to a time literal.
```

Comment:
Should this comment be amended ? Perhaps something like: "Check if `call` compares a date/time column with a date/time constant. Otherwise just treat it like a regular binary operator."

Reply:
done.